snuba/admin/static/runtime_config/auditlog.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactNode, useState } from "react";
+
+import Client from "../api_client";
+import { containerStyle, paragraphStyle } from "./styles";
+import { Table } from "../table";
+import { ConfigValue } from "./types";
+
+function AuditLog(props: { api: Client }) {
+  const { api } = props;
+  const [data, setData] = useState<any[] | null>(null);
```

Comment:
Yes, will fix.

Reply:
Actually this is fixed in the other PR that comes after this one which addresses a bunch of type issues on both the UI and API https://github.com/getsentry/snuba/pull/2194. Will leave it as any here to avoid merging issues.